Modules/Map/QuestieMap.lua

```diff
@@ -378,17 +358,18 @@ function QuestieMap:ShowNPC(npcID, icon, scale, title, body, disableShiftToRemov
     data.IsObjectiveNote = false
     data.ManualTooltipData = {}
     data.ManualTooltipData.Title = title or (npc.name .. " (" .. l10n("NPC") .. ")")
+    local WeaponMasterSkills = require("WeaponMasterSkills")
     if Questie.IsTBC or Questie.IsWotlk then
         -- Blood Elf Starting Area Weapon Trainers
-        weaponMasterSkills[16621] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
-        weaponMasterSkills[17005] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
+        WeaponMasterSkills.data[16621] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
+        WeaponMasterSkills.data[17005] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
         -- Draenei Starting Area Weapon Trainers
-        weaponMasterSkills[16773] = {"Crossbows", "Daggers", "One-Handed Maces", "One-Handed Swords", "Two-Handed Maces", "Two-Handed Swords"}
+        WeaponMasterSkills.data[16773] = {"Crossbows", "Daggers", "One-Handed Maces", "One-Handed Swords", "Two-Handed Maces", "Two-Handed Swords"}
     end
     if npc.subName == "Weapon Master" then
-        local skills = weaponMasterSkills[data.id]
+        local skills = WeaponMasterSkills.data[data.id]
         if skills then
-            data.ManualTooltipData.Title = appendSkillsToTitle(data.ManualTooltipData.Title, skills)
+            data.ManualTooltipData.Title = WeaponMasterSkills.appendSkillsToTitle(data.ManualTooltipData.Title, skills)
         end
```

Comment:
We should remove the `if` int this place and simply call `WeaponMasterSkills.AppendSkillsToTitle`. If we don't have skills for a specific NPC then log it on warning level and return an empty string (aka nothing is appended).